### PR TITLE
Fix pserve argument ordering.

### LINKED ISSUE #1092: Make serve recipe is failing
Seems related to #1087
```
gsurita-30820:kinto gsurita$ make serve 
.venv/bin/kinto --ini config/kinto.ini init
Select the backend you would like to use: (1 - postgresql, 2 - redis, default - memory) 
INFO   Created config /Users/gsurita/kinto/config/kinto.ini
.venv/bin/kinto --ini config/kinto.ini migrate
/Users/gsurita/kinto/kinto/core/initialization.py:567: UserWarning: HTTPS is not enabled
  warnings.warn('HTTPS is not enabled')
INFO   Running kinto 6.0.0.dev0. 
echo '{"name":"kinto","version":"5.3.1-118-g5fc2dedf","source":"https://github.com/gabisurita/kinto/","commit":"5fc2dedfc0a75d9f337ccdbcc935fa051f377164"}' > version.json
.venv/bin/kinto --ini config/kinto.ini start --reload
usage: kinto [-h] [-n NAME] [-s SERVER_TYPE] [--server-name SECTION_NAME]
             [--reload] [--reload-interval RELOAD_INTERVAL] [-b] [-v] [-q]
             [config_uri] [config_vars [config_vars ...]]
kinto: error: unrecognized arguments: http_port=8888
make: *** [serve] Error 2
```

### PR DESCRIPTION
**Pull request by Natim**
Refs: Pylons/pyramid#2962

Fixes #1092

- [x] Add documentation.
- [x] Add tests.
- [x] Add a changelog entry.

### CODE REVIEW

File: kinto/__main__.py [hunk 1]
```diff
@@ -145,11 +145,14 @@ def main(args=None):
...
     elif which_command == 'start':
-        pserve_argv = ['pserve', config_file]
+        pserve_argv = ['pserve']
         if parsed_args['reload']:
             pserve_argv.append('--reload')
+        pserve_argv.append('-v')
+        pserve_argv.append(config_file)
         pserve_argv.append('http_port={}'.format(parsed_args['port']))
-        pserve.main(pserve_argv)
+        print(pserve_argv)
```
Comment (leplatrem) [inline kinto/__main__.py]: left over or useful info?

File: kinto/__main__.py [hunk 2]
```diff
@@ -145,11 +145,14 @@ def main(args=None):
                                          parsed_args['collection'])
 
     elif which_command == 'start':
-        pserve_argv = ['pserve', config_file]
+        pserve_argv = ['pserve']
         if parsed_args['reload']:
             pserve_argv.append('--reload')
+        pserve_argv.append('-v')
```
Comment (leplatrem) [inline kinto/__main__.py]: should we add this only if our verbose option is set?

**Comment by glasserc** [comment]
The fact that the argument order is important seems like a bug; should we report it upstream?

**Comment by Natim** [comment]
> The fact that the argument order is important seems like a bug; should we report it upstream?

We have the exact same issue with kinto and subcommands.

The command help explicits it so it doesn't seems like a bug to me.

File: tests/test_main.py [hunk 3]
```diff
@@ -135,6 +135,26 @@ def test_cli_start_runs_pserve(self):
...
+
+    def test_cli_start_with_verbose_option_runs_pserve_with_verbose(self):
+        with mock.patch('kinto.__main__.pserve.main') as mocked_pserve:
+            res = main(['--ini', TEMP_KINTO_INI, 'init',
+                        '--backend', 'memory'])
+            assert res == 0
+            res = main(['--ini', TEMP_KINTO_INI, 'start', '-v'])
+            assert res == 0
+            assert mocked_pserve.call_count == 1
+            assert mocked_pserve.call_args_list[0][1]['argv'][1] == '-v'
```
Comment (Natim) [inline tests/test_main.py]: We could probably use call_args instead of call_args_list.

**Pull request merged**